src/index.html

```diff
@@ -4,9 +4,374 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="/" class="nav__item is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/"
+            class="nav__item"
+            data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Фото
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="catalog__cards">
+          <div class="card catalog__card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+              class="card__img"
+            >
+            <p class="card__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </p>
+            <p class="card__code">
+              Код товара: 195434
+            </p>
+            <div class="card__starsWrapper">
+              <div class="stars stars--four">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__feedback">
+                Отзывов: 5
+              </p>
+            </div>
+            <div class="card__priceWrapper">
+              <p class="card__priceTitle">
+                Цена:
+              </p>
+              <p class="card__price">
+                69 999 грн
+              </p>
+            </div>
+            <a href="/"
+               class="card__button"
+               data-qa="card-hover"
+               >
+              Купить
+            </a>
+          </div>
+          <div class="card catalog__card" >
```

Comment:
again suggestion extra space)

Reply:
@SerhiiZiatiuk Thanks, I've corrected it.